submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+$primaryColor: #000;
+$mainFont: 'Merriweather', serif;
+$btnColor: #416aff;
+$bcgColor: #ffffff;
+
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  min-height: 100%;
+  font-family: $mainFont;
+  font-size: 16px;
+  line-height: 1.71;
+  letter-spacing: 0.03em;
+  color: $primaryColor;
+  background-image: url('../popup-icons/ua.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+h1,
+h2,
+h3,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: currentColor;
+}
+
+.list {
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.visually-hidden {
+  position: absolute;
+  clip: rect(1px 1px 1px 1px);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
+
+.wrapper {
+  display: grid;
+  grid-template: minmax(100px, auto) 1fr / 1 fr;
+}
+
+.header {
+  padding: 15px 30px;
+  display: grid;
+  grid-template: 1fr / 1fr minmax(100px, auto) minmax(100px, auto);
+  justify-content: end;
+  background-color: transparent;
+  border-bottom: 2px solid rgb(163, 165, 167);
+  column-gap: 30px;
+
+  @media screen and (min-width: 768px) {
```

Comment:
Please, don't use `@media` inside properties, it's really hardly to read. Move it underneath in one separate block with all properties, like so:
` @media screen and (min-width: 768px) {
    .header {
         column-gap: 50px;
                }
...
}`

Reply:
ok